src/integrators/integrator_utils.jl

```diff
@@ -237,40 +237,45 @@ function stepsize_controller!(integrator,alg::Union{StandardControllerAlgs,
   q
 end
 function step_accept_controller!(integrator,alg::Union{StandardControllerAlgs,
-                                 OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Standard}},q) where {CS, AD}
+  OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Standard}},q) where {CS, AD}
   integrator.dt/q # dtnew
 end
 function step_reject_controller!(integrator,alg::Union{StandardControllerAlgs,
-                                 OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Standard}}) where {CS, AD}
+  OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Standard}}) where {CS, AD}
   integrator.dt = integrator.qold
 end
 
-function stepsize_controller!(integrator,
-                              alg::OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Predictive}) where {CS, AD}
+const PredictiveControllerAlgs = Union{RKC}
+function stepsize_controller!(integrator,alg::Union{PredictiveControllerAlgs,
+    OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,:Predictive}}) where {CS, AD}
 
   # Gustafsson predictive stepsize controller
 
   if iszero(integrator.EEst)
     q = inv(integrator.opts.qmax)
   else
     gamma = integrator.opts.gamma
-    if alg isa RadauIIA5
-      @unpack nl_iters = integrator.cache
-      @unpack max_iter = alg
+    if alg isa RKC
```

Comment:
@ChrisRackauckas @YingboMa Added this condition because earlier the calls were designed for Implicit methods but except choosing fac, rest of the mechanics was exactly same as described in the paper

Reply:
that works